app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,51 @@
+<%  if current_user.present? %>
+  <nav class="navbar navbar-expand-sm navbar-dark bg-dark">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample03" aria-controls="navbarsExample03" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarsExample03">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item">
+          <%= link_to 'Home', root_path, class: 'nav-link' %>
+        </li>
+        <li class="nav-item">
+          <%= link_to 'Notes', user_notes_path(current_user.id), class: 'nav-link' %>
```

Comment:
We need consistency while using strings. It could be single quoted or double quoted every where.

Reply:
done